cwms-data-api/src/main/java/cwms/cda/data/dao/JooqDao.java

```diff
@@ -209,7 +209,9 @@ public static Condition caseInsensitiveLikeRegex(Field<String> field, String reg
         return new CustomCondition() {
             @Override
             public void accept(org.jooq.Context<?> ctx) {
-                if (ctx.family() == ORACLE) {
+                if("*".equals(regex) || ".*".equals(regex)) {
+                    ctx.visit(DSL.trueCondition());
```

Comment:
I think this needs some comment about why this branch is necessary.  Why does it come first, why is trueCondition the right thing etc.   I think you are making "*" or ".*" also match null - is that what this is doing? Is that needed?  It looked like you were explicitly handling null designators another way.   Adding extra branches here makes me nervous.

Reply:
sorry, this was an initial attempt, but should get reverted in favor of actually handling null designators.

Reply:
reverted this back